pkg/sysfs/utils.go

```diff
@@ -101,13 +103,13 @@ func writeSysfsEntry(base, entry string, val, oldp interface{}, args ...interfac
 	path := filepath.Join(base, entry)
 
 	switch val.(type) {
-	case string, ID, int, uint, int8, uint8, int16, uint16, int32, uint32, int64, uint64:
```

Comment:
Hmm, don't we need to handle an `idset.ID` here to keep the code functionally identical ?

Reply:
Note that commit https://github.com/intel/goresctrl/commit/7d764492dc971b5704626dc50527ebfa6ef8cb38 in goresctrl changed the type of ID
```
-type ID int
+type ID = int
```
so ID is now int which means that there cannot be duplicate int type checks various switches.

Reply:
Yeah, I defined ID that way in goresctrl in order to reduce the need for pointless (IMO) type casts

Reply:
Ah, ok. I missed that completely.